### PR TITLE
added Zookeeper to Patroni chart

### PR DESCRIPTION
**Pull request by ghost**
Added Zookeeper option

### CODE REVIEW

**Comment by ghost** [comment]
Contributor License Agreement signed

**Comment by thelinuxfoundation** [comment]
bumping for CLA checks

**Comment by ghost** [comment]
re adding GitHub account to CLA

**Comment by ghost** [comment]
added secondary email account to github

**Comment by hjacobs** [comment]
LGTM, thanks for the PR! :smile:

**Comment by prydonius** [comment]
@linki ping

does it make sense to add zookeeper as a dependency to the chart so it can be installed with it?

**Comment by linki** [comment]
@prydonius I think it does. Maybe we should do something similar to https://github.com/kubernetes/charts/pull/1089/files#diff-c87e703f9d61597f6f671b11cb326e55R44?

**Comment by unguiculus** [comment]
Marking this as stale. Please update during the next week.

**Comment by linki** [comment]
@Andre-Freitas can you have another look at this? Adding zookeeper as a dependecy makes sense if it's configured.

**Comment by ghost** [comment]
@linki i can try to check this but in our environment we have our zookeeper outside of k8s. I need to check in helm dependency it i have anyway to differentiate from a k8s or local zookeeper instance

**Comment by ghost** [comment]
@linki I have added an option to have Etcd or Zookeeper outside of k8s cluster. I have made a few tests and its working

**Comment by unguiculus** [comment]
ping @linki

File: incubator/patroni/values.yaml [hunk 1]
```diff
@@ -25,7 +25,15 @@ Credentials:
   Admin: cola
   Standby: pinacolada
 
-# Patroni depends on etcd, configure it here
+## Distribution Configuration stores. Please note that only one of the following stores should be selected.
 Etcd:
-  Host: # fill-in value for etcd host (etcd.default.svc.cluster.local), leave blank to use the discovery parameter
+  Enable: true
```
Comment (sstarcher) [inline incubator/patroni/values.yaml]: Should these follow the recommendations under - https://github.com/kubernetes/helm/blob/9cfdbd1f7773aff6d3de0080796fc10559b91948/docs/chart_best_practices/requirements.md#conditions-and-tags

**Comment by ghost** [comment]
@linki fixed

**Comment by linki** [comment]
Thanks @Andre-Freitas

File: incubator/patroni/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: patroni
 description: 'Highly available elephant herd: HA PostgreSQL cluster.'
-version: 0.4.0
+version: 0.4.1
```
Comment (linki) [inline incubator/patroni/Chart.yaml]: Let's bump this to `0.5` because we add a feature.

**Pull request merged**